utilities.ts

```diff
@@ -20,8 +20,8 @@ var inBrowser = typeof console !== "undefined" && console.info;
 declare var putstr;
 declare var printErr;
 
-/** @const */ var release: boolean = true;
-/** @const */ var profile: boolean = true;
+/** @const */ var release: boolean = jsGlobal.config.release;
+/** @const */ var profile: boolean = !release;
```

Comment:
@mbebenita mentioned that doing this will prevent Closure from compiling out the various assertions that are prefixed with `release ||`, so we should figure out some different way to apply the value of RELEASE here. Maybe we can move these declarations to a separate config.ts file that we include in j2me.js by adding it to references.ts.